### DISCUSSION TITLE
Primitive parsing

### DISCUSSION THREAD

**Discussion by QuentinWidlocher** [airjp73/rvf#56] [Q&A] [answered]
Hi, this is really a great tool, but I miss only one thing for my use case.

Are you considering adding the parsing of primitives like booleans, dates, numbers etc. ?

It could be really useful to implement it. I use it myself with a tool similar to Validated Forms and it makes me save a lot of time.

**Comment by airjp73** [comment, marked as answer]
Hi! I'm glad you like it 😃 

I'm not entirely sure what you mean. Are you thinking of something like [zod-form-data](https://www.remix-validated-form.io/zod-form-data)? Could you give me an example of what you're ideal use-case looks like?

**Reply by QuentinWidlocher** [reply]
I just saw zod-form-data and it's precisely what I wanted to achieve, it's just that I didn't see it done like that.
It works really nicely, thanks !
